src/lib/viewers/controls/media/MediaSettingsControls/_styles.scss

```diff
@@ -0,0 +1,48 @@
+@import '../styles';
+
+$padding: 8px;
+
+@mixin bp-MediaSettingsMenuItem {
```

Comment:
Do these need to be separate mixins or can they just be defined in the MediaSettingsMenuItem.scss?